test/OWNERS

```diff
@@ -6,3 +6,14 @@ approvers:
 reviewers:
   - cluster-api-reviewers
   - cluster-api-test-reviewers
+
+filters:
+  ".*":
+    labels:
+    - area/testing
+  "*infrastructure/docker/*":
+    labels:
+    - area/infrastructure-docker
+  "*e2e*":
+    labels:
+    - area/e2e
```

Comment:
Probably makes more sense to just have `area/e2e` for everything in this directory. IMO `area/testing` should be for stuff outside e2e tests. `area/infrastructure-docker` should just be for the docker provider itself. The manifests in this folder are really part of the e2e tests.

Reply:
agreed